pages/common/git-rebase.md

```diff
@@ -17,3 +17,7 @@
 - Abort a rebase in-progress:
 
 `git rebase --abort`
+
+- Rebase your local branch by specifying new base commit and old base commit:
+
+`git rebase --onto {{NEW-BASE-COMMIT}} {{OLD-BASE-COMMIT}}`
```

Comment:
Exactly right @liuderchi. And a full-repo scan to fix tokens would be most welcome!